src/components/screens/ScoreSetCreator.vue

```diff
@@ -95,15 +95,25 @@
                       </div>
                       <div v-else style="position: relative;">
                         <span class="p-float-label">
-                          <Dropdown
+                          <AutoComplete
+                            ref="experimentInput"
                             v-model="experiment"
+                            dropdown
                             :id="$scopedId('input-experiment')"
+                            :suggestions="experimentSuggestionsList"
```

Comment:
I would suggest using `editableExperiments` here directly. The suggestions list is more for the autocomplete Vue component, and the dropdown is much better at handling empty lists. This will be useful for another comment below.

Reply:
I changed it to dropdown with filtering function. It looks better. Autocomplete has a small bug with the dropdown button.